fmriprep/cli/parser.py

```diff
@@ -401,6 +401,14 @@ def _slice_time_ref(value, parser):
         help="Output individual echo time series with slice, motion and susceptibility "
         "correction. Useful for further Tedana processing post-fMRIPrep.",
     )
+    g_outputs.add_argument(
+        "--aggregate-session-reports",
```

Comment:
If you want the config file to pick it up automatically, the mapped name should match that in the config file:

```suggestion
        "--aggregate-session-reports",
        dest="aggr_ses_reports",
```